.vscode/launch.json

```diff
@@ -41,6 +41,20 @@
           "remoteRoot": "/usr/src/homeassistant"
         }
       ]
+    },
+    {
+      // settings that VS Code uses when debugging tests
+      // See https://code.visualstudio.com/docs/python/testing#_debug-tests
+      "name": "Python: Debug Tests",
```

Comment:
Nice! (Although the build-in test tooling already provides this...

Reply:
This does not add a new task for debugging, check the docs link I added. This lets you define settings for when the Python extension does debug. So, you can have different pytest settings for debugging and not debugging. So when you debug a test, it disabled xdist and pytest-cov, but if you run them (no debug) it uses both.